### DISCUSSION TITLE
Self-signed TLS/SSL certificates and tools *not* using the system CA store by default

### DISCUSSION THREAD

**Discussion by chopeen** [renovatebot/renovate#22297] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

The document [self-hosting.md?plain=1#L440](https://github.com/renovatebot/renovate/blob/main/docs/usage/examples/self-hosting.md?plain=1#L440) explains that Node comes with an own certificate authority store, but it does not mention that Node is not the only tool like that. Similar customization is required for the `pip` utility and Python packages like `requests` or `urllib`.

Would you accept a PR extending the `Dockerfile` to include these Python customizations?

**Comment by rarkins** [comment]
If it's a quick change, please submit the PR so we can look. I don't know exactly what you mean. Also, it might turn out to be better to make changes in github.com/containerbase/base

**Reply by chopeen** [reply]
There you go - https://github.com/renovatebot/renovate/pull/22299. Please have a look.
